src/cosmoz-tree.ts

```diff
@@ -3,7 +3,27 @@ Navigator through object with treelike datastructure and default settings.
 
 @demo demo/index.html
 */
+
+export type TreeOptions = {
```

Comment:
Just a preference but i'd name this simply `Options` because we are already in the context of comoz-tree.